hknweb/settings/common.py

```diff
@@ -78,6 +78,8 @@
                 'django.contrib.messages.context_processors.messages',
                 'social_django.context_processors.backends',
                 'social_django.context_processors.login_redirect',
+                'social_django.context_processors.login_redirect',
```

Comment:
Why is the same line twice?

Reply:
Uh I commented OOPS on this line but apparently forgot to publish it